src/porepy/fracs/fractures_2d.py

```diff
@@ -1,17 +1,19 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
 """
-Created on Mon Dec 10 09:24:49 2018
-
-@author: Eirik Keilegavlens
+Module contains class for representing a fracture network in a 2d domain.
 """
 import numpy as np
 import logging
 import networkx as nx
 import csv
```

Comment:
do we want to change the name of the file? I mean, to be consistent with the 3d? (or better change the 3d file name)